src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -30,11 +30,17 @@ import {
 import { i18n } from '@kbn/i18n';
 import { groupBy, sortBy } from 'lodash';
 import React, { useRef } from 'react';
+import * as Rx from 'rxjs';
+import { useObservable } from 'react-use';
 import { AppCategory } from '../../../../types';
 import { OnIsLockedUpdate } from './';
-import { NavLink, RecentNavLink } from './nav_link';
+import { ChromeNavLink, ChromeRecentlyAccessedHistoryItem } from '../..';
+import { createRecentNavLink, createEuiListItem } from './nav_link';
+import { HttpStart } from '../../../http';
+import { InternalApplicationStart } from '../../../application/types';
+import './collapsible_nav.scss';
```

Comment:
I'm not sure importing SCSS will work properly right now in Core. Reason being that Core is built by the new and legacy webpack configurations and the legacy one does not support scss imports. You could test this by going to the Maps application (which is still in legacy)

Reply:
<img width="608" alt="Screen Shot 2020-05-19 at 16 35 37" src="https://user-images.githubusercontent.com/4188087/82375577-6be63c00-99e6-11ea-8d58-c89f5de14b01.png">
Still works in maps 👍

Reply:
Nit: I've been moving these imports to the top of the file (first import) so it's way more obvious that this component relies on external styles.